config/operatorhub/seldondeployment.crd.yaml

```diff
@@ -0,0 +1,140 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
why is the  CRD defined twice. Here and later on in deploy folder. Same with roles.

Reply:
The config/operatorhub has been removed. And the CRD submitted in the operatorhub community repo has been updated to include the full JSON.